extensions/openrouter-models-finder/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "openrouter-models-finder",
+  "title": "OpenRouter Models Finder",
+  "description": "Search and copy AI model IDs from OpenRouter to clipboard",
+  "icon": "icon.png",
+  "author": "jeejeeguan",
+  "license": "MIT",
+  "version": "1.0.0",
```

Comment:
See https://developers.raycast.com/information/versioning.

We don't use the `version` field for versioning. Let's remove it.